tests/functional/resolvers.ts

```diff
@@ -544,6 +554,29 @@ describe("Resolvers", () => {
         expect(itemType.name).toEqual("String");
       });
 
+      it("should generate explicit array of nullable string return type for query", async () => {
+        const explicitNullableItemArrayQuery = getQuery("explicitNullableItemArrayQuery");
+        const type = explicitNullableItemArrayQuery.type as IntrospectionNonNullTypeRef;
+        const listType = type.ofType as IntrospectionListTypeRef;
+        const itemType = listType.ofType as IntrospectionNamedTypeRef;
+
+        expect(listType.kind).toEqual(TypeKind.LIST);
```

Comment:
Please check also the `type`, like for fields:
`expect(nonNullArrayType.kind).toEqual(TypeKind.NON_NULL);`